python/kvikio/nvcomp.py

```diff
@@ -0,0 +1,224 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
+# See file LICENSE for terms.
+
+from enum import Enum
+
+import cupy as cp
+import numpy as np
+
+import kvikio._lib.pynvcomp as _lib
+
+
+def cp_to_nvcomp_dtype(in_type: cp.dtype) -> Enum:
+    """ Convert np/cp dtypes to nvcomp integral dtypes.
+
+    Returns
```

Comment:
Missing a parameters section, which would answer my above query.

Reply:
I added one that might satisfy your curiosity.